src/Landing/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import FacebookButton from "../auth/socialAuth/facebook/components/Facebook";
+import GoogleButton from "../auth/socialAuth/google/components/Google";
+import axios from "axios";
+import history from '../commons/history';
+
+export default class SocialAuthentication extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      errorMessage: false
```

Comment:
Give this boolean a better name... something that would let us know else where in the code that we're probably dealing with a boolean